### ISSUE TITLE
PHP 7.3 compact() E_NOTICE

### ISSUE THREAD

**Issue by ericcholis** [UnionOfRAD/lithium#1342]
In php 7.3, compat now throws an `E_NOTICE` when passing a variable that isn't set: [From the docs](https://www.php.net/manual/en/function.compact.php):

````
compact() now issues an E_NOTICE level error if a given string refers to an unset variable. Formerly, such strings have been silently skipped.
````

I'm having a hard time detecting this with PHPCompatibility checks. Seems like there is an [open PHPCompatibility](https://github.com/PHPCompatibility/PHPCompatibility/issues/704) issue that details the difficulty a bit more.

I'll take a crack at cleaning this up as it comes up.

**Comment by mariuswilms** [comment]
Hey I've begun working on PHP 7.3 compat for the upcoming 1.2 release in https://github.com/UnionOfRAD/lithium/tree/support-php-7.3

Currently some extract command test cases are failing, I didn't yet look at the integration tests.

PHP 7.3 is generally not yet supported.

**Comment by mariuswilms** [comment]
Yes, you're right. I just implemented support for 7.3 in 1.2.
